packages/components/src/Lettera.purs

```diff
@@ -282,6 +284,22 @@ getLatest start limit paper = do
                                 <> "&paper=" <> Paper.toString paper
                               )
 
+getByDay :: Int -> Int -> Date -> Paper -> Aff (LetteraResponse (Array ArticleStub))
+getByDay start limit date paper = do
+  let formattedDate = show (fromEnum (year date)) <> "-"
```

Comment:
I'd recommend using `Data.Formatter.DateTime`. The month and day fields expect two digits anyway and it'll fail if you'd try something like `2023-1-1` on it.